### PR TITLE
apply code highlighting to WCF docs 

### PR DESCRIPTION
**Pull request by johnmwright**
## Summary

This is a continuation of https://github.com/dotnet/docs/pull/4991

Add language designation (c#, svc etc) to code blocks in the following portions of the WCF docs:

[WCF :: Basic :: Services](https://docs.microsoft.com/en-us/dotnet/framework/wcf/samples/services)
[WCF :: Basic :: Routing Services](https://docs.microsoft.com/en-us/dotnet/framework/wcf/samples/routing-services)
[WCF :: :Basic :: Management](https://docs.microsoft.com/en-us/dotnet/framework/wcf/samples/management)
[WCF :: Basic: :: Discovery](https://docs.microsoft.com/en-us/dotnet/framework/wcf/samples/discovery-samples) 
[WCF :: Basic :: Contact](https://docs.microsoft.com/en-us/dotnet/framework/wcf/samples/contract)

### CODE REVIEW

File: docs/framework/wcf/samples/known-types.md [hunk 1]
```diff
@@ -110,14 +110,14 @@ public class DataContractCalculatorService : IDataContractCalculator
...
+```csharp
 // Create a client  
 DataContractCalculatorClient client =   
     new DataContractCalculatorClient();  
   
 // Call the Add service operation.  
-ComplexNumber value1 = new ComplexNumber(); value1.real = 1; value1.imaginary = 2;  
-ComplexNumber value2 = new ComplexNumber(); value2.real = 3; value2.imaginary = 4;  
+ComplexNumber value1 = new ComplexNumber() { real = 1, imaginary = 2 };  
+ComplexNumber value2 = new ComplexNumber() { real = 3, imaginary = 4 };
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/known-types.md]: This is a nice catch! Thanks for both fixing the formatting and modernizing this code!

File: docs/framework/wcf/samples/namedpipe-activation.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ Press <ENTER> to terminate client.
         > [!NOTE]
         >  This command is a single line of text.  
   
-         This command enables the /servicemodelsamples application to be accessed using both `http://localhost/servicemodelsamples` and `net.tcp://localhost/servicemodelsamples`.  
+         This command enables the /servicemodelsamples application to be accessed using both [http://localhost/servicemodelsamples](http://localhost/servicemodelsamples) and `net.tcp://localhost/servicemodelsamples`.
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/namedpipe-activation.md]: The original is correct. Please revert this change.

File: docs/framework/wcf/samples/security-validation.md [hunk 3]
```diff
@@ -13,12 +13,12 @@ This sample demonstrates how to use a custom behavior to validate services on a
 ## Endpoint Validation Custom Behavior  
  By adding user code to the `Validate` method contained in the <xref:System.ServiceModel.Description.IServiceBehavior> interface, custom behavior can be given to a service or endpoint to perform user-defined actions. The following code is used to loop through each endpoint contained in a service, which searches through their binding collections for secure bindings.  
   
-```  
+```csharp
 public void Validate(ServiceDescription serviceDescription,   
                                        ServiceHostBase serviceHostBase)  
 {  
     // Loop through each endpoint individually gathering their
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/security-validation.md]: nit unrelated to your PR: there should be a comma after individually

File: docs/framework/wcf/samples/security-validation.md [hunk 4]
```diff
@@ -13,12 +13,12 @@ This sample demonstrates how to use a custom behavior to validate services on a
...
  By adding user code to the `Validate` method contained in the <xref:System.ServiceModel.Description.IServiceBehavior> interface, custom behavior can be given to a service or endpoint to perform user-defined actions. The following code is used to loop through each endpoint contained in a service, which searches through their binding collections for secure bindings.  
   
-```  
+```csharp
 public void Validate(ServiceDescription serviceDescription,   
                                        ServiceHostBase serviceHostBase)  
 {  
     // Loop through each endpoint individually gathering their    
-       binding elements.  
+    // binding elements.
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/security-validation.md]: good catch!

File: docs/framework/wcf/samples/tcp-activation.md [hunk 5]
```diff
@@ -169,7 +169,7 @@ Press <ENTER> to terminate client.
         ```  
   
         > [!NOTE]
-        > This command is a single line of text. This command enables the /servicemodelsamples application to be accessed using both `http://localhost/servicemodelsamples` and `net.tcp://localhost/servicemodelsamples`.  
+        > This command is a single line of text. This command enables the /servicemodelsamples application to be accessed using both [http://localhost/servicemodelsamples](http://localhost/servicemodelsamples) and `net.tcp://localhost/servicemodelsamples`.
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/tcp-activation.md]: The original version is correct; please revert the change.

File: docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md [hunk 6]
```diff
@@ -52,7 +52,7 @@ svcutil.exe /n:http://Microsoft.ServiceModel.Samples,Microsoft.ServiceModel.Samp
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md]: the console language identifier can be added to line 49.

File: docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md [hunk 7]
```diff
@@ -52,7 +52,7 @@ svcutil.exe /n:http://Microsoft.ServiceModel.Samples,Microsoft.ServiceModel.Samp
   
  This class must be included in a project and the project should be configured to generate a COM-visible, signed assembly when compiled. The following attribute should be included in the AssemblyInfo.cs file.  
   
-```  
+```csharp
 [assembly: ComVisible(true)]  
 ```
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md]: The console identifier can be added to lines 61 and 67

File: docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md [hunk 8]
```diff
@@ -73,7 +73,7 @@ gacutil.exe /i client.dll
   
  ComCalcClient.vbs client application uses the `GetObject` function to construct a proxy for the service, using the service moniker syntax to specify the address, binding, and contract for the service.  
   
-```  
+```vb
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md]: This isn't Visual Basic, its VBScript. I don't think a language identifier for VBScript is supported. There shouldn't be a language identifier here. This is also true of lines 92, 104,137, 149, and 172.

File: docs/framework/wcf/samples/wcf-analytic-tracing.md [hunk 9]
```diff
@@ -47,7 +47,7 @@ This sample demonstrates how to add your own tracing events into the stream of a
   
 4.  Run the WCF test client (WcfTestClient.exe).  
   
-     The WCF test client (WcfTestClient.exe) is located in the \<Visual Studio 2012 Install Dir>\Common7\IDE\ WcfTestClient.exe (default Visual Studio 2012 install dir is C:\Program Files\Microsoft Visual Studio 10.0).  
+     The WCF test client (WcfTestClient.exe) is located at `\<Visual Studio 2012 Install Dir>\Common7\IDE\WcfTestClient.exe` (default Visual Studio 2012 install dir is `C:\Program Files\Microsoft Visual Studio 10.0`).
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/wcf-analytic-tracing.md]: nit unrelated to your PR: The clause in parentheses should be a separate sentence -- e.g., "(The default Visual Studio 2012 install directory is...).

File: docs/framework/wcf/samples/wcf-services-and-event-tracing-for-windows.md [hunk 10]
```diff
@@ -20,15 +20,15 @@ This sample demonstrates how to use the analytic tracing in Windows Communicatio
 
      In the Web browser, click **Calculator.svc**. The URI of the WSDL document for the service should appear in the browser. Copy that URI.
 
-     By default, the service starts listening for requests on port 1378 (http://localhost:1378/Calculator.svc).
+     By default, the service starts listening for requests on port 1378 [http://localhost:1378/Calculator.svc](http://localhost:1378/Calculator.svc).
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/wcf-services-and-event-tracing-for-windows.md]: Instead of a link, the URL should be fenced.

File: docs/framework/wcf/samples/wcf-services-and-event-tracing-for-windows.md [hunk 11]
```diff
@@ -20,15 +20,15 @@ This sample demonstrates how to use the analytic tracing in Windows Communicatio
...
 
 4.  Run the WCF test client (WcfTestClient.exe).
 
-     The WCF test client (WcfTestClient.exe) is located in the \<Visual Studio 2012 Install Dir>\Common7\IDE\ WcfTestClient.exe (default Visual Studio 2012 install dir is C:\Program Files\Microsoft Visual Studio 10.0).
+     The WCF test client (WcfTestClient.exe) is located at `\<Visual Studio 2012 Install Dir>\Common7\IDE\WcfTestClient.exe` (default Visual Studio 2012 install dir is `C:\Program Files\Microsoft Visual Studio 10.0`).
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/wcf-services-and-event-tracing-for-windows.md]: same comment as line 50 of the previous article.

File: docs/framework/wcf/samples/wcf-services-and-event-tracing-for-windows.md [hunk 12]
```diff
@@ -20,15 +20,15 @@ This sample demonstrates how to use the analytic tracing in Windows Communicatio
...
 
 5.  Within the WCF test client, add the service by selecting **File**, and then **Add Service**.
 
-     Add the endpoint address in the input box. The default is `http://localhost:1378/Calculator.svc`.
+     Add the endpoint address in the input box. The default is [http://localhost:1378/Calculator.svc](http://localhost:1378/Calculator.svc).
```
Comment (rpetrusha) [inline docs/framework/wcf/samples/wcf-services-and-event-tracing-for-windows.md]: The original is correct. Please revert this change.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @johnmwright, for adding the missing language identifiers, as well as for some of the other catches that you've made to improve code and formatting. I've left a number of comments for you to address.

**Comment by johnmwright** [comment]
@rpetrusha I’ll make those changes soon and update the PR.

One note: in my previous PR, it was specifically requested to use `svc` for those files, so I used it here to be consistent. https://github.com/dotnet/docs/pull/4991#discussion_r183850464

File: docs/framework/wcf/samples/iis-hosting-using-inline-code.md [hunk 13]
```diff
@@ -41,9 +41,10 @@ This sample demonstrates how to implement a service hosted by Internet Informati
   
  The service implementation calculates and returns the appropriate result.  
   
-```  
+```svc
```
Comment (Thraka) [inline docs/framework/wcf/samples/iis-hosting-using-inline-code.md]: Technically this is just an html file. There is no language highlighter for svc

File: docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md (hunk 8 above)
Comment (Thraka) [reply docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md]: VBScript is a supported highlight language

Comment (rpetrusha) [reply docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md]: Is its language identifier `vbscript`, @thraka?

**Review by Thraka — APPROVED** [review]

**Review by rpetrusha — APPROVED** [review]

File: docs/framework/wcf/samples/msmq-activation.md [hunk 14]
```diff
@@ -85,13 +85,13 @@ public class OrderProcessorService : IOrderProcessor
   
  The Service.svc file itself contains a directive to create the `OrderProcessorService`.  
   
-```xml  
+```svc
```
[the remark this answers is not in the document]
Comment (rpetrusha) [reply docs/framework/wcf/samples/msmq-activation.md]: Leaving svc is fine @johnmwright. Thanks for making the extra set of changes. I'll merge your PR now.
